knife-supermarket.gemspec

```diff
@@ -14,5 +14,5 @@ Gem::Specification.new do |s|
   s.homepage    = 'https://github.com/cwebberOps/knife-supermarket'
   s.license       = 'Apache 2.0'
   s.require_paths = ["lib"]
-  s.add_dependency "chef", ">= 0.10.10"
+  s.add_dependency "chef", ">= 0.10.10", "<= 12.11.18"
```

Comment:
Would the observed behavior in this case (chef 12.12.x, knife-supermarket gem installed) be `knife supermarket` commands do nothing but output the dep notice advising the user to uninstall the gem? (Because the overridden `run` methods would be NOP at that point.)